app/components/badge-bitbucket-pipelines.js

```diff
@@ -0,0 +1,16 @@
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName: 'span',
+  classNames: ['badge'],
+  repository: alias('badge.attributes.repository'),
+  branch: computed('badge.attributes.branch', function() {
+    return encodeURIComponent(this.get('badge.attributes.branch'));
```

Comment:
the URL encoding is mandatory (e.g. [this link](https://bitbucket.org/delan/iyypdgarimwgp/addon/pipelines/home#!/results/branch/branch/name/with/slashes/page/1) doesn’t work)